---
title: A paper a day
layout: post
comments: true
---

In an effort to read more papers this year, I'm going to read a paper (or something paper-like) each day for the remainder of the year, and post each 
paper below as I go.

### Dec 22

Nelson MG, Linheiro RS, Bergman CM. McClintock: An integrated pipeline for detecting transposable element insertions in whole genome shotgun sequencing data. bioRxiv. 2016. [doi:10.1101/095372](http://dx.doi.org/10.1101/095372)

### Dec 21

Vidalis A, Živković D, Wardenaar R, Roquis D, Tellier A, Johannes F. Methylome evolution in plants. Genome Biology. 2016. [doi:10.1186/s13059-016-1127-5](http://dx.doi.org/10.1186/s13059-016-1127-5)

### Dec 20

Park K, Kim MY, Vickers M, Park J-S, Hyun Y, Okamoto T, et al. DNA demethylation is initiated in the central cells of Arabidopsis and rice. PNAS. 2016. [doi:10.1073/pnas.1619047114](http://dx.doi.org/10.1073/pnas.1619047114)

<!--break-->

### Dec 19

Jaitin DA, Weiner A, Yofe I, Lara-Astiaso D, Keren-Shaul H, David E, et al. Dissecting Immune Circuits by Linking CRISPR-Pooled Screens with Single-Cell RNA-Seq. Cell. 2016. [doi:10.1016/j.cell.2016.11.039](http://dx.doi.org/10.1016/j.cell.2016.11.039)

### Dec 18

Baruzzo G, Hayer KE, Kim EJ, Di Camillo B, FitzGerald GA, Grant GR. Simulation-based comprehensive benchmarking of RNA-seq aligners. Nature Methods. 2016. [doi:10.1038/nmeth.4106](http://dx.doi.org/10.1038/nmeth.4106)

### Dec 17

Stoiber MH, Quick J, Egan R, Lee JE, Celniker SE, Neely R, et al. De novo Identification of DNA Modifications Enabled by Genome-Guided Nanopore Signal Processing. bioRxiv. 2016. [doi:10.1101/094672](http://dx.doi.org/10.1101/094672)

### Dec 16

Nagano T, Lubling Y, Varnai C, Dudley C, Leung W, Baran Y, et al. Cell cycle dynamics of chromosomal organisation at single-cell resolution. bioRxiv. 2016. [doi:10.1101/094466](http://dx.doi.org/10.1101/094466)

### Dec 15

Poplin R, Newburger D, Dijamco J, Nguyen N, Loy D, Gross S, et al. Creating a universal SNP and small indel variant caller with deep neural networks. bioRxiv. 2016. [doi:10.1101/092890](http://dx.doi.org/10.1101/092890)

### Dec 14

D'haeseleer P. How does gene expression clustering work? Nature Biotechnology. 2005. [doi:10.1038/nbt1205-1499](http://dx.doi.org/10.1038/nbt1205-1499)

### Dec 13

Halperin E, Stephan DA. SNP imputation in association studies. Nature Biotechnology. 2009. [doi:10.1038/nbt0409-349](http://dx.doi.org/10.1038/nbt0409-349)

### Dec 12

Charlesworth B, Charlesworth D. Darwin and genetics. Genetics. 2009. [doi:10.1534/genetics.109.109991](http://dx.doi.org/10.1534/genetics.109.109991)

### Dec 11

van Dijk PJ, Ellis THN. The Full Breadth of Mendel's Genetics. Genetics. 2016. [doi:10.1534/genetics.116.196626](http://dx.doi.org/10.1534/genetics.116.196626)

### Dec 10

Lio C-W, Zhang J, González-Avalos E, Hogan PG, Chang X, Rao A. Tet2 and Tet3 cooperate with B-lineage transcription factors to regulate DNA modification and chromatin accessibility. eLife. 2016. [doi:10.7554/eLife.18290](http://dx.doi.org/10.7554/eLife.18290)

### Dec 9

Pawluk A, Amrani N, Zhang Y, Garcia B, Hidalgo-Reyes Y, Lee J, et al. Naturally Occurring Off-Switches for CRISPR-Cas9. Cell. 2016. [doi:10.1016/j.cell.2016.11.017](http://dx.doi.org/10.1016/j.cell.2016.11.017)

### Dec 8

Buckley AR, Standish KA, Bhutani K, Ideker T, Harismendy O, Carter H, et al. Pan-Cancer Analysis Reveals Technical Artifacts in The Cancer Genome Atlas (TCGA) Germline Variant Calls. bioRxiv. 2016. [doi:10.1101/092163](http://dx.doi.org/10.1101/092163)

### Dec 7

Efroni I, Mello A, Nawy T, Ip P-L, Rahni R, DelRose N, et al. Root Regeneration Triggers an Embryo-like Sequence Guided by Hormonal Interactions. Cell. 2016. [doi:10.1016/j.cell.2016.04.046](http://dx.doi.org/10.1016/j.cell.2016.04.046)

### Dec 6

Yang N, MacArthur DG, Gulbin JP, Hahn AG, Beggs AH, Easteal S, et al. ACTN3 genotype is associated with human elite athletic performance. The American Journal of Human Genetics. 2003. [doi:10.1086/377590](http://dx.doi.org/10.1086/377590)

### Dec 5

Noble WS. How does multiple testing correction work? Nature Biotechnology. 2009. [doi:10.1038/nbt1209-1135](http://dx.doi.org/10.1038/nbt1209-1135)

### Dec 4

Williamson CHD, Sanchez A, Vazquez A, Gutman J, Sahl JW. Bacterial genome reduction as a result of short read sequence assembly. bioRxiv. 2016. [doi:10.1101/091314](http://dx.doi.org/10.1101/091314)

### Dec 3

Nattestad M, Chin C-S, Schatz MC. Ribbon: Visualizing complex genome alignments and structural variation. bioRxiv. 2016. [doi:10.1101/082123](http://dx.doi.org/10.1101/082123)

### Dec 2

Carvalho AB, Dupim EG, Goldstein G. Improved assembly of noisy long reads by k-mer validation. Genome Research. 2016. [doi:10.1101/gr.209247.116](http://dx.doi.org/10.1101/gr.209247.116)

### Dec 1

Lu Z, Hofmeister BT, Vollmers C, DuBois RM, Schmitz RJ. Combining ATAC-seq with nuclei sorting for discovery of cis-regulatory regions in plant genomes. Nucleic Acids Research. 2016. [doi:10.1093/nar/gkw1179](http://dx.doi.org/10.1093/nar/gkw1179)

### Nov 30

Weinreb C, Wolock S, Klein A. SPRING: a kinetic interface for visualizing high dimensional single-cell expression data. bioRxiv. 2016. [doi:10.1101/090332](http://dx.doi.org/10.1101/090332)

### Nov 29

Vergara Z, Sequeira-Mendes J, Morata J, Hénaff E, Peiró R, Costas C, et al. Retrotransposons are specified as DNA replication origins in the gene-poor regions of Arabidopsis heterochromatin. bioRxiv. 2016. [doi:10.1101/090183](http://dx.doi.org/10.1101/090183)

### Nov 28

Wagner A, Regev A, Yosef N. Revealing the vectors of cellular identity with single-cell genomics. 2016. Nature Biotechnology. [doi:10.1038/nbt.3711](http://dx.doi.org/10.1038/nbt.3711)

### Nov 27

http://genomestake.blogspot.com.au/2016/11/thirty-fifth-take-is-what-ails-us.html

### Nov 26

Gasch AP, Hose J, Newton MA, Sardi M, Yong M, Wang Z. Further support for aneuploidy tolerance in wild yeast and effects of dosage compensation on gene copy-number evolution. eLife. 2016. [doi:10.7554/eLife.14409](http://dx.doi.org/10.7554/eLife.14409)

### Nov 25

Li Z, Wang S, Cheng J, Su C, Zhong S, Liu Q, et al. Intron Lariat RNA Inhibits MicroRNA Biogenesis by Sequestering the Dicing Complex in Arabidopsis. Mittelsten Scheid O, editor. PLoS Genetics. 2016. [doi:10.1371/journal.pgen.1006422](http://dx.doi.org/10.1371/journal.pgen.1006422)

### Nov 24

la Rica de L, Deniz Ö, Cheng KCL, Todd CD, Cruz C, Houseley J, et al. TET-dependent regulation of retrotransposable elements in mouse embryonic stem cells. Genome Biology. 2016. [doi:10.1186/s13059-016-1096-8](http://dx.doi.org/10.1186/s13059-016-1096-8)

### Nov 23

Stegle O, Teichmann SA, Marioni JC. Computational and analytical challenges in single-cell transcriptomics. Nature Reviews Genetics. 2015. [doi:10.1038/nrg3833](http://dx.doi.org/10.1038/nrg3833)

### Nov 22

Chuong EB, Elde NC, Feschotte C. Regulatory activities of transposable elements: from conflicts to benefits. 2016. Nature Reviews Genetics. [doi:10.1038/nrg.2016.139](http://dx.doi.org/10.1038/nrg.2016.139)

### Nov 21

Tirosh I, Venteicher AS, Hebert C, Escalante LE, Patel AP, Yizhak K, et al. Single-cell RNA-seq supports a developmental hierarchy in human oligodendroglioma. 2016. Nature. [doi:10.1038/nature20123](http://dx.doi.org/10.1038/nature20123)

### Nov 20

Sulak M, Fong L, Mika K, Chigurupati S, Yon L, Mongan NP, et al. TP53 copy number expansion is associated with the evolution of increased body size and an enhanced DNA damage response in elephants. 2016. eLife. [doi:10.7554/eLife.11994](http://dx.doi.org/10.7554/eLife.11994)

### Nov 19

L Lun AT, Bach K, Marioni JC. Pooling across cells to normalize single-cell RNA sequencing data with many zero counts. Genome Biology. 2016. [doi:10.1186/s13059-016-0947-7](http://dx.doi.org/10.1186/s13059-016-0947-7)

### Nov 18

Barau J, Teissandier A, Zamudio N, Roy S, Nalesso V, Hérault Y, et al. The DNA methyltransferase DNMT3C protects male germ cells from transposon activity. Science. 2016. [doi:10.1126/science.aah5143](http://dx.doi.org/10.1126/science.aah5143)

### Nov 17

Molinaro AM, Pearson BJ. In silico lineage tracing through single cell transcriptomics identifies a neural stem cell population in planarians. Genome Biology. 2016. [doi:10.1186/s13059-016-0937-9](http://dx.doi.org/10.1186/s13059-016-0937-9)

### Nov 16

Ntranos V, Kamath GM, Zhang JM, Pachter L, Tse DN. Fast and accurate single-cell RNA-seq analysis by clustering of transcript-compatibility counts. Genome Biology. 2016. [doi:10.1186/s13059-016-0970-8](http://dx.doi.org/10.1186/s13059-016-0970-8)

### Nov 15

Li S, Yamada M, Han X, Ohler U, Benfey PN. High-Resolution Expression Map of the Arabidopsis Root Reveals Alternative Splicing and lincRNA Regulation. Dev Cell. 2016. [doi:10.1016/j.devcel.2016.10.012](http://dx.doi.org/10.1016/j.devcel.2016.10.012)

### Nov 14

Korthauer KD, Chu L-F, Newton MA, Li Y, Thomson J, Stewart R, et al. A statistical approach for identifying differential distributions in single-cell RNA-seq experiments. Genome Biology. 2016. [doi:10.1186/s13059-016-1077-y](http://dx.doi.org/10.1186/s13059-016-1077-y)

### Nov 13

Bendall SC, Davis KL, Amir E-AD, Tadmor MD, Simonds EF, Chen TJ, et al. Single-cell trajectory detection uncovers progression and regulatory coordination in human B cell development. Cell. 2014. [doi:10.1016/j.cell.2014.04.005](http://dx.doi.org/10.1016/j.cell.2014.04.005)

### Nov 12

Ding B, Zheng L, Zhu Y, Li N, Jia H, Ai R, et al. Normalization and noise reduction for single cell RNA-seq experiments. Bioinformatics. 2015. [doi:10.1093/bioinformatics/btv122](http://dx.doi.org/10.1093/bioinformatics/btv122)

### Nov 11

Grün D, van Oudenaarden A. Design and Analysis of Single-Cell Sequencing Experiments. Cell. 2015. [doi:10.1016/j.cell.2015.10.039](http://dx.doi.org/10.1016/j.cell.2015.10.039)

### Nov 10

Buenrostro JD, Giresi PG, Zaba LC, Chang HY, Greenleaf WJ. Transposition of native chromatin for fast and sensitive epigenomic profiling of open chromatin, DNA-binding proteins and nucleosome position. Nature Methods. 2013. [doi:10.1038/nmeth.2688](http://dx.doi.org/10.1038/nmeth.2688)

### Nov 9

Buettner F, Natarajan KN, Casale FP, Proserpio V, Scialdone A, Theis FJ, et al. Computational analysis of cell-to-cell heterogeneity in single-cell RNA-sequencing data reveals hidden subpopulations of cells. Nature Biotechnology. 2015. [doi:10.1038/nbt.3102](http://dx.doi.org/10.1038/nbt.3102)

### Nov 8

Satgé C, Moreau S, Sallet E, Lefort G, Auriac M-C, Remblière C, et al. Reprogramming of DNA methylation is critical for nodule development in Medicago truncatula. Nature Plants. [doi:10.1038/nplants.2016.166](http://dx.doi.org/10.1038/nplants.2016.166)

### Nov 7

Kolodziejczyk AA, Kim JK, Svensson V, Marioni JC, Teichmann SA. The Technology and Biology of Single-Cell RNA Sequencing. Molecular Cell. 2015. [doi:10.1016/j.molcel.2015.04.005](http://dx.doi.org/10.1016/j.molcel.2015.04.005)

### Nov 6

Svensson V, Natarajan KN, Ly L-H, Miragaia RJ, Labalette C, Macaulay IC, et al. Power Analysis of Single Cell RNA‐Sequencing Experiments. bioRxiv. 2016. [doi:10.1101/073692](http://dx.doi.org/10.1101/073692)

### Nov 5

Bacher R, Kendziorski C. Design and computational analysis of single-cell RNA-sequencing experiments. Genome Biology. 2016. [doi:10.1186/s13059-016-0927-y](http://dx.doi.org/10.1186/s13059-016-0927-y)

### Nov 4

Song L, Huang S-SC, Wise A, Castanon R, Nery JR, Chen H, et al. A transcription factor hierarchy defines an environmental stress response network. Science. 2016. [doi:10.1126/science.aag1550](http://dx.doi.org/10.1126/science.aag1550)

### Nov 3

Liu S, Trapnell C. Single-cell transcriptome sequencing: recent advances and remaining challenges. F1000Res. 2016, [doi:10.12688/f1000research.7223.1](http://dx.doi.org/10.12688/f1000research.7223.1)

### Nov 2

Wattenberg M, Viégas F, Johnson I. How to Use t-SNE Effectively.  2016. [http://distill.pub/2016/misread-tsne/](http://distill.pub/2016/misread-tsne/)

### Nov 1

Faridani OR, Abdullayev I, Hagemann-Jensen M, Schell JP, Lanner F, Sandberg R. Single-cell sequencing of the small-RNA transcriptome. Nat Biotech. 2016. [doi:10.1038/nbt.3701](http://dx.doi.org/10.1038/nbt.3701)

### Oct 31

Olsson A, Venkatasubramanian M, Chaudhri VK, Aronow BJ, Salomonis N, Singh H, et al. Single-cell analysis of mixed-lineage states leading to a binary cell fate choice. Nature. 2016. [doi:10.1038/nature19348](http://dx.doi.org/10.1038/nature19348)

### Oct 30

Yi X, Dean AM. Phenotypic plasticity as an adaptation to a functional trade-off. eLife. [doi:10.7554/eLife.19307](http://dx.doi.org/10.7554/eLife.19307)

### Oct 29

Efroni I, Birnbaum KD. The potential of single-cell profiling in plants. Genome Biology. 2016. [doi:10.1186/s13059-016-0931-2](http://dx.doi.org/10.1186/s13059-016-0931-2)

### Oct 28

Trizzino M, Park Y, Holsbach-Beltrame M, Aracena K, Mika K, Caliskan M, et al. Transposable element exaptation is the primary source of novelty in the primate gene regulatory landscape. bioRxiv. 2016. [doi:10.1101/083980](http://dx.doi.org/10.1101/083980)

### Oct 27

Lockton S, Gaut BS. The evolution of transposable elements in natural populations of self-fertilizing Arabidopsis thaliana and its outcrossing relative Arabidopsis lyrata. BMC Evolutionary Biology. 2010. [doi:10.1186/1471-2148-10-10](http://dx.doi.org/10.1186/1471-2148-10-10)

### Oct 26

Freeling M, Lyons E, Pedersen B, Alam M, Ming R, Lisch D. Many or most genes in Arabidopsis transposed after the origin of the order Brassicales. Genome Research. 2008. [doi:10.1101/gr.081026.108](http://dx.doi.org/10.1101/gr.081026.108)

### Oct 25

Lynch M, Ackerman MS, Gout J-F, Long H, Sung W, Thomas WK, et al. Genetic drift, selection and the evolution of the mutation rate. Nature Reviews Genetics. 2016. [doi:10.1038/nrg.2016.104](http://dx.doi.org/10.1038/nrg.2016.104)

### Oct 24

di Iulio J, Bartha I, Wong E, Yu H-C, Hicks M, Shah N, et al. The human functional genome defined by genetic diversity. bioRxiv. 2016. [doi:10.1101/082362](http://dx.doi.org/10.1101/082362)

### Oct 23

Greaves IK, Eichten SR, Groszmann M, Wang A, Ying H, Peacock WJ, et al. Twenty-four–nucleotide siRNAs produce heritable trans-chromosomal methylation in F1 Arabidopsis hybrids. PNAS. 2016. [doi:10.1073/pnas.1613623113](http://dx.doi.org/10.1073/pnas.1613623113)

### Oct 22

Watson JM, Platzer A, Kazda A, Akimcheva S, Valuchova S, Nizhynska V, et al. Germline replications and somatic mutation accumulation are independent of vegetative life span in Arabidopsis. PNAS. 2016. [doi:10.1073/pnas.1609686113](http://dx.doi.org/10.1073/pnas.1609686113)

### Oct 21

Schmidt F, Gasparoni N, Gasparoni G, Gianmoena K, Cadenas C, Polansky JK, et al. Combining transcription factor binding affinities with open-chromatin data for accurate gene expression prediction. bioRxiv. 2016. [doi:10.1101/081935](http://dx.doi.org/10.1101/081935)

### Oct 20

Huff JT, Zilberman D, Roy SW. Mechanism for DNA transposons to generate introns on genomic scales. Nature. [doi:10.1038/nature20110](http://dx.doi.org/10.1038/nature20110)

### Oct 19

Schalk C, Drevensek S, Kramdi A, Kassam M, Ahmed I, Cognat V, et al. DNA DAMAGE BINDING PROTEIN2 Shapes the DNA Methylation Landscape. The Plant Cell. 2016. [doi:10.1105/tpc.16.00474](http://dx.doi.org/10.1105/tpc.16.00474)

### Oct 18

Hikabe O, Hamazaki N, Nagamatsu G, Obata Y, Hirao Y, Hamada N, et al. Reconstitution in vitro of the entire cycle of the mouse female germ line. Nature. 2016. [doi:10.1038/nature20104](http://dx.doi.org/10.1038/nature20104)

### Oct 17

Pietzenuk B, Markus C, Gaubert H, Bagwan N, Merotto A, Bucher E, et al. Recurrent evolution of heat-responsiveness in Brassicaceae COPIA elements. Genome Biology. 2016. [doi:10.1186/s13059-016-1072-3](http://dx.doi.org/10.1186/s13059-016-1072-3)

### Oct 16

D'haeseleer P. How does DNA sequence motif discovery work? Nature Biotechnology. 2006. [doi:10.1038/nbt0806-959](http://dx.doi.org/10.1038/nbt0806-959)

### Oct 15

Trapnell C, Salzberg SL. How to map billions of short reads onto genomes. Nature Biotechnology. 2009. [doi:10.1038/nbt0509-455](http://dx.doi.org/10.1038/nbt0509-455)


### Oct 14

Do CB, Batzoglou S. What is the expectation maximization algorithm? Nature Biotechnology. 2008. [doi:10.1038/nbt1406](http://dx.doi.org/10.1038/nbt1406)

### Oct 13

D'haeseleer P. What are DNA sequence motifs? Nature Biotechnology. 2006. [doi:10.1038/nbt0406-423](http://dx.doi.org/10.1038/nbt0406-423)

### Oct 12

Novikova PY, Hohmann N, Nizhynska V, Tsuchimatsu T, Ali J, Muir G, et al. Sequencing of the genus Arabidopsis identifies a complex history of nonbifurcating speciation and abundant trans-specific polymorphism. Nat Gen. 2016. [doi:10.1038/ng.3617](http://dx.doi.org/10.1038/ng.3617)

### Oct 11

Urban JM, Bliss J, Lawrence CE, Gerbi SA. Sequencing ultra-long DNA molecules with the Oxford Nanopore MinION. bioRxiv. 2015. [doi:10.1101/019281](http://dx.doi.org/10.1101/019281)

### Oct 10

Bousios A, Gaut BS, Darzentas N. Considerations and complications of mapping small RNA libraries to transposable elements. bioRxiv. 2016. [doi:10.1101/079749](http://dx.doi.org/10.1101/079749)

### Oct 9

McLeay RC, Lesluyes T, Cuellar Partida G, Bailey TL. Genome-wide in silico prediction of gene expression. Bioinformatics. 2012. [doi:10.1093/bioinformatics/bts529](http://dx.doi.org/10.1093/bioinformatics/bts529)

### Oct 8

Cheng C, Gerstein M. Modeling the relative relationship of transcription factor binding and histone modifications to gene expression levels in mouse embryonic stem cells. Nucleic Acids Research. 2012. [doi:10.1093/nar/gkr752](http://dx.doi.org/10.1093/nar/gkr752)

### Oct 7

Miska EA, Ferguson-Smith AC. Transgenerational inheritance: Models and mechanisms of non–DNA sequence–based inheritance. Science. 2016. [doi:10.1126/science.aaf4945](http://dx.doi.org/10.1126/science.aaf4945)

### Oct 6

Ito H, Yoshida T, Tsukahara S, Kawabe A. Evolution of the ONSEN retrotransposon family activated upon heat stress in Brassicaceae. Gene. 2013. [doi:10.1016/j.gene.2013.01.034](http://dx.doi.org/10.1016/j.gene.2013.01.034)

### Oct 5

Budden DM, Hurley DG, Crampin EJ. Predictive modelling of gene expression from transcriptional regulatory elements. Briefings in Bioinformatics. 2015. [doi:10.1093/bib/bbu034](http://dx.doi.org/10.1093/bib/bbu034)

### Oct 4

Matsunaga W, Ohama N, Tanabe N, Masuta Y, Masuda S, Mitani N, et al. A small RNA mediated regulation of a stress-activated retrotransposon and the tissue specific transposition during the reproductive period in Arabidopsis. Front Plant Sci. 2015. [doi:10.3389/fpls.2015.00048](http://dx.doi.org/10.3389/fpls.2015.00048)

### Oct 3

Larivière V, Costas R. How Many Is Too Many? On the Relationship between Research Productivity and Impact. PLoS ONE. 2016. [doi:10.1371/journal.pone.0162709](http://dx.doi.org/10.1371/journal.pone.0162709)

### Oct 2

Kingsford C, Salzberg SL. What are decision trees? Nature Biotechnology. 2008. [doi:10.1038/nbt0908-1011](http://dx.doi.org/10.1038/nbt0908-1011)

### Oct 1

Olah C, Carter S. Attention and Augmented Recurrent Neural Networks. 2016. [http://distill.pub/2016/augmented-rnns/](http://distill.pub/2016/augmented-rnns/)

### Sep 30

Cavrak VV, Lettner N, Jamge S, Kosarewicz A, Bayer LM, Mittelsten Scheid O. How a Retrotransposon Exploits the Plant's Heat Stress Response for Its Activation. PLoS Genetics. 2014. [doi:10.1371/journal.pgen.1004115](http://dx.doi.org/10.1371/journal.pgen.1004115)

### Sep 29

Brzezinka K, Altmann S, Czesnick H, Nicolas P, Gorka M, Benke E, et al. Arabidopsis FORGETTER1 mediates stress-induced chromatin memory through nucleosome remodeling. eLife. 2016. [doi:10.7554/eLife.17061](http://dx.doi.org/10.7554/eLife.17061)

### Sep 28

Herman JJ, Sultan SE. DNA methylation mediates genetic variation for adaptive transgenerational plasticity. Proceedings of the Royal Society of London B: Biological Sciences. 2016. [doi:10.1098/rspb.2016.0988](http://dx.doi.org/10.1098/rspb.2016.0988)

### Sep 27

Burkart-Waco D, Ngo K, Dilkes B, Josefsson C, Comai L. Early disruption of maternal-zygotic interaction and activation of defense-like responses in Arabidopsis interspecific crosses. The Plant Cell; 2013. [doi:10.1105/tpc.112.108258](http://dx.doi.org/10.1105/tpc.112.108258)

### Sep 26

Burkart-Waco D, Ngo K, Lieberman M, Comai L. Perturbation of parentally biased gene expression during interspecific hybridization. PLoS ONE. 2015. [doi:10.1371/journal.pone.0117293](http://dx.doi.org/10.1371/journal.pone.0117293)

### Sep 25

Eddy SR. What is dynamic programming? Nature Biotechnology. 2004. [doi:10.1038/nbt0704-909](http://dx.doi.org/10.1038/nbt0704-909)

### Sep 24

Malaspinas A-S, Westaway MC, Muller C, Sousa VC, Lao O, Alves I, et al. A genomic history of Aboriginal Australia. Nature. 2016. [doi:10.1038/nature18299](http://dx.doi.org/10.1038/nature18299)

### Sep 23

Wulfridge P, Langmead B, Feinberg AP, Hansen K. Choice of reference genome can introduce massive bias in bisulfite sequencing data. bioRxiv. 2016. [doi:10.1101/076844](http://dx.doi.org/10.1101/076844)

### Sep 22

Yang G, Nagel DH, Feschotte C, Hancock CN, Wessler SR. Tuned for Transposition: Molecular Determinants Underlying the Hyperactivity of a Stowaway MITE. Science. 2009. [doi:10.1126/science.1175688](http://dx.doi.org/10.1126/science.1175688)

### Sep 21

Libbrecht MW, Noble WS. Machine learning applications in genetics and genomics. Nature Reviews Genetics. 2015. [doi:10.1038/nrg3920](http://dx.doi.org/10.1038/nrg3920)

### Sep 20

Huang X, Yang S, Gong J, Zhao Q, Feng Q, Zhan Q, et al. Genomic architecture of heterosis for yield traits in rice. Nature. 2016. [doi:10.1038/nature19760](http://dx.doi.org/10.1038/nature19760)

### Sep 19

Compeau PEC, Pevzner PA, Tesler G. How to apply de Bruijn graphs to genome assembly. Nature Biotechnology. 2011. [doi:10.1038/nbt.2023](http://dx.doi.org/10.1038/nbt.2023)

### Sep 18

Price MN, Wetmore KM, Waters RJ, Callaghan M, Ray J, Kuehl JV, et al. Deep Annotation of Protein Function across Diverse Bacteria from Mutant Phenotypes. bioRxiv. 2016. [doi:10.1101/072470](http://dx.doi.org/10.1101/072470)

### Sep 17

Eddy SR. What is a hidden Markov model? Nature Biotechnology. 2004. [doi:10.1038/nbt1004-1315](http://dx.doi.org/10.1038/nbt1004-1315)

### Sep 16

Krogh A. What are artificial neural networks? Nature Biotechnology. 2008. [doi:10.1038/nbt1386](http://dx.doi.org/10.1038/nbt1386)

### Sep 15

Noble WS. What is a support vector machine? Nature Biotechnology. 2006. [doi:10.1038/nbt1206-1565](http://dx.doi.org/10.1038/nbt1206-1565)

### Sep 14

Chen J, Wrightsman T, Wessler SR, Stajich JE. RelocaTE2: a high resolution transposable element polymorphism mapping tool for population resequencing. PeerJ Preprints. 2016. [doi:10.7287/peerj.preprints.2447v1](http://dx.doi.org/10.7287/peerj.preprints.2447v1)

### Sep 13

Baker M, Why scientists must share their research code. Nature. 2016. [doi:10.1038/nature.2016.20504](http://dx.doi.org/10.1038/nature.2016.20504)

### Sep 12

Sindi SS, Önal S, Peng LC, Wu H-T, Raphael BJ. An integrative probabilistic model for identification of structural variation in sequencing data. Genome Biology. 2012. [doi:10.1186/gb-2012-13-3-r22](http://dx.doi.org/10.1186/gb-2012-13-3-r22)

### Sep 11

Spies N, Weng Z, Bishara A, McDaniel J, Catoe D, Zook JM, et al. Genome-wide reconstruction of complex structural variants using read clouds. bioRxiv. 2016. [doi:10.1101/074518](http://dx.doi.org/10.1101/074518)

### Sep 10

Li X, Kim Y, Tsang EK, Davis JR, Damani FN, Chiang C, et al. The impact of rare variation on gene expression across tissues. bioRxiv. 2016. [doi:10.1101/074443](http://dx.doi.org/10.1101/074443)

### Sep 9

Peng RD. Reproducible Research in Computational Science. Science. 2011. [doi:10.1126/science.1213847](http://dx.doi.org/10.1126/science.1213847)

### Sep 8

[http://simplystatistics.org/open%20science/2016/09/07/interview-with-a-data-sucker/](http://simplystatistics.org/open%20science/2016/09/07/interview-with-a-data-sucker/)

### Sep 7

Kover PX, Valdar W, Trakalo J, Scarcelli N, Ehrenreich IM, Purugganan MD, et al. A Multiparent Advanced Generation Inter-Cross to Fine-Map Quantitative Traits in Arabidopsis thaliana. PLoS Genet. 2009. [doi:10.1371/journal.pgen.1000551](http://dx.doi.org/10.1371/journal.pgen.1000551)

### Sep 6

Geman D, Geman S. Opinion: Science in the age of selfies. PNAS. 2016. [doi:10.1073/pnas.1609793113](http://dx.doi.org/10.1073/pnas.1609793113)

### Sep 5

Muñoz-Gómez SA, Roger AJ. Leaving negative ancestors behind. eLife. 2016. [doi:10.7554/eLife.20061](http://dx.doi.org/10.7554/eLife.20061)

### Sep 4

Altman N, Krzywinski M. Points of significance: Sources of variation. Nat Meth. 2015. [doi:10.1038/nmeth.3224](http://dx.doi.org/10.1038/nmeth.3224)

### Sep 3

Blainey P, Krzywinski M, Altman N. Points of Significance: Replication. Nat Meth. 2014. [doi:10.1038/nmeth.3091](http://dx.doi.org/10.1038/nmeth.3091)

### Sep 2

Wilson G, Bryan J, Cranston K, Kitzes J, Nederbragt L, Teal TK. Good Enough Practices in Scientific Computing. arXiv. 2016. [arXiv:1609.00037](http://arxiv.org/abs/1609.00037v1) 

### Sep 1

Harris EY, Ounit R, Lonardi S. BRAT-nova: fast and accurate mapping of bisulfite-treated reads. Bioinformatics. 2016. [doi:10.1093/bioinformatics/btw226](http://dx.doi.org/10.1093/bioinformatics/btw226)

### Aug 31

Schnorrenberg S, Grotjohann T, Vorbrüggen G, Herzig A, Hell SW, Jakobs S. In vivo super-resolution RESOLFT microscopy of Drosophila melanogaster. eLife. 2016. [doi:10.7554/eLife.15567](http://dx.doi.org/10.7554/eLife.15567)

### Aug 30

Martin AR, Gignoux CR, Walters RK, Wojcik GL, Gravel S, Daly MJ, et al. Population genetic history and polygenic risk biases in 1000 Genomes populations. bioRxiv. 2016. [doi:10.1101/070797](http://dx.doi.org/10.1101/070797)

### Aug 29

Wright SI, Gaut BS. Molecular population genetics and the search for adaptive evolution in plants. Molecular Biology and Evolution. 2005. [doi:10.1093/molbev/msi035](http://dx.doi.org/10.1093/molbev/msi035)

### Aug 28

Maumus F, Quesneville H. Ancestral repeats have shaped epigenome and genome composition for millions of years in Arabidopsis thaliana. Nature Communications. 2014. [doi:10.1038/ncomms5104](http://dx.doi.org/10.1038/ncomms5104)

### Aug 27

Takuno S, Ran J-H, Gaut BS. Evolutionary patterns of genic DNA methylation vary across land plants. Nature Plants. 2016. [doi:10.1038/nplants.2015.222](http:///dx.doi.org/10.1038/nplants.2015.222)

### Aug 26

[http://simplystatistics.org/2016/08/24/replication-crisis/](http://simplystatistics.org/2016/08/24/replication-crisis/)


### Aug 25

Altman DG, Bland JM. How to obtain the confidence interval from a P value. BMJ. 2011. [doi:10.1136/bmj.d2090](http://dx.doi.org/10.1136/bmj.d2090)

### Aug 24

Hollister JD, Gaut BS. Epigenetic silencing of transposable elements: a trade-off between reduced transposition and deleterious effects on neighboring gene expression. Genome Research. 2009. [doi:10.1101/gr.091678.109](http://dx.doi.org/10.1101/gr.091678.109)

### Aug 23

Cano-Rodriguez D, Gjaltema RAF, Jilderda LJ, Jellema P, Dokter-Fokkens J, Ruiters MHJ, et al. Writing of H3K4Me3 overcomes epigenetic silencing in a sustained but context-dependent manner. Nature Communications. 2016. [doi:10.1038/ncomms12284](http://dx.doi.org/10.1038/ncomms12284)

### Aug 22

Yang D-L, Zhang G, Tang K, Li J, Yang L, Huang H, et al. Dicer-independent RNA-directed DNA methylation in Arabidopsis. Cell Res. 2016. [doi:10.1038/cr.2015.145](http://dx.doi.org/10.1038/cr.2015.145)

### Aug 21

Walley JW, Sartor RC, Shen Z, Schmitz RJ, Wu KJ, Urich MA, et al. Integration of omic networks in a developmental atlas of maize. Science. 2016. [doi:10.1126/science.aag1125](http://dx.doi.org/10.1126/science.aag1125)

### Aug 20

Elyashiv E, Sattath S, Hu TT, Strutsovsky A, McVicker G, Andolfatto P, et al. A Genomic Map of the Effects of Linked Selection in Drosophila. PLoS Genetics. 2016. [doi:10.1371/journal.pgen.1006130](http://dx.doi.org/10.1371/journal.pgen.1006130)

### Aug 19

Keren L, Hausser J, Lotan-Pompan M, Vainberg Slutskin I, Alisar H, Kaminski S, et al. Massively Parallel Interrogation of the Effects of Gene Expression Levels on Fitness. Cell. 2016. [doi:10.1016/j.cell.2016.07.024](http://dx.doi.org/10.1016/j.cell.2016.07.024)

### Aug 18

Wei B, Liu H, Liu X, Xiao Q, Wang Y, Zhang J, et al. Genome-wide characterization of non-reference transposons in crops suggests non-random insertion. BMC Genomics. [doi:10.1186/s12864-016-2847-3](http://dx.doi.org/10.1186/s12864-016-2847-3)

### Aug 17

Weir W, Capewell P, Foth B, Clucas C, Pountain A, Steketee P, et al. Population genomics reveals the origin and asexual evolution of human infective trypanosomes. eLife. 2016. [doi:10.7554/eLife.11473](http://dx.doi.org/10.7554/eLife.11473)

### Aug 16

Altman N, Krzywinski M. Points of Significance: Association, correlation and causation. Nat Meth. [doi:10.1038/nmeth.3587](http://dx.doi.org/10.1038/nmeth.3587)

### Aug 15

Altman N, Krzywinski M. Points of Significance: Simple linear regression. Nat Meth. [doi:10.1038/nmeth.3627](http://dx.doi.org/10.1038/nmeth.3627)

### Aug 14

Garalde DR, Snell EA, Jachimowicz D, Heron AJ, Bruce M, Lloyd J, et al. Highly parallel direct RNA sequencing on an array of nanopores. bioRxiv. 2016. [doi:10.1101/068809](http://dx.doi.org/10.1101/068809)

### Aug 13

Merenciano M, Ullastres A, de Cara MAR, Barrón MG, González J. Multiple Independent Retroelement Insertions in the Promoter of a Stress Response Gene Have Variable Molecular and Functional Effects in Drosophila. PLoS Genet. 2016. [doi:10.1371/journal.pgen.1006249](http://dx.doi.org/10.1371/journal.pgen.1006249)

### Aug 12

Kulesa A, Krzywinski M, Blainey P, Altman N. Points of Significance: Sampling distributions and the bootstrap. Nat Meth. 2015. [doi:10.1038/nmeth.3414](http://dx.doi.org/10.1038/nmeth.3414)

### Aug 11

Puga JL, Krzywinski M, Altman N. Points of Significance: Bayesian networks. Nat Meth. 2015. [doi:10.1038/nmeth.3550](http://dx.doi.org/10.1038/nmeth.3550)

### Aug 10

Panda K, Ji L, Neumann DA, Daron J, Schmitz RJ, Slotkin RK. Full-length autonomous transposable elements are preferentially targeted by expression-dependent forms of RNA-directed DNA methylation. Genome Biology. 2016. [doi:10.1186/s13059-016-1032-y](http://dx.doi.org/10.1186/s13059-016-103)

### Aug 9

Dickson, A. Introduction to Graph Theory. Utah Math. 2006. [pdf](https://www.math.utah.edu/mathcircle/notes/MC_Graph_Theory.pdf)

### Aug 8

Rowan BA, Patel V, Weigel D, Schneeberger K. Rapid and Inexpensive Whole-Genome Genotyping-by-Sequencing for Crossover Localization and Fine-Scale Genetic Mapping. G3: Genes \| Genomes \| Genetics. 2015. [doi:10.1534/g3.114.016501](http://dx.doi.org/10.1534/g3.114.016501) 

### Aug 7

Puga JL, Krzywinski M, Altman N. Points of significance: Bayesian statistics. Nat Meth. 2015. [doi:10.1038/nmeth.3368](http://dx.doi.org/10.1038/nmeth.3368)

### Aug 6

Eddy SR. What is Bayesian statistics? Nature Biotechnology. 2004. [doi:10.1038/nbt0904-1177](http://dx.doi.org/10.1038/nbt0904-1177)

### Aug 5

Arribas-Hernández L, Marchais A, Poulsen C, Haase B, Hauptmann J, Benes V, et al. The Slicer Activity of ARGONAUTE1 Is Required Specifically for the Phasing, Not Production, of Trans-Acting Short Interfering RNAs in Arabidopsis. The Plant Cell. 2016. [DOI: 10.1105/tpc.16.00121](http://dx.doi.org/10.1105/tpc.16.00121)

### Aug 4

Li Q, Yuan W, Wang X, Liu Y, Ma X, Sun H, et al. A pair of transposon‐derived proteins regulate active DNA demethylation in Arabidopsis. EMBO. 2016. [doi:10.15252/embj.201694284](http//:dx.doi.org/10.15252/embj.201694284)

### Aug 3

Patil P, Peng RD, Leek J. A statistical definition for reproducibility and replicability. bioRxiv. 2016. [doi:10.1101/066803](http://dx.doi.org/10.1101/066803)

### Aug 2

Wickham H. The split-apply-combine strategy for data analysis. Journal of Statistical Software. [2011](https://www.jstatsoft.org/article/view/v040i01)

### Aug 1

Vanderkam D, Aksoy BA, Hodes I, Perrone J, Hammerbacher J. pileup.js: a JavaScript library for interactive and in-browser visualization of genomic data. Bioinformatics. 2016. [doi:10.1093/bioinformatics/btw167](http://dx.doi.org/10.1093/bioinformatics/btw167)

### Jul 31

Puga JL, Krzywinski M, Altman N. Points of Significance: Bayes' theorem. Nat Meth. 2015. [doi:10.1038/nmeth.3335](http://dx.doi.org/10.1038/nmeth.3335)

### Jul 30

Wang J, Meng X, Yuan C, Harrison AP, Chen M. The roles of cross-talk epigenetic patterns in Arabidopsis thaliana. Briefings in Functional Genomics. 2016. [doi:10.1093/bfgp/elv025](http://dx.doi.org/10.1093/bfgp/elv025)

### Jul 29

Halsey LG, Curran-Everett D, Vowler SL, Drummond GB. The fickle P value generates irreproducible results. Nat Meth. 2016. [doi:10.1038/nmeth.3288](http://dx.doi.org/10.1038/nmeth.3288)

van Helden J. Confidence intervals are no salvation from the alleged fickleness of the P value. Nat Meth.2016. [doi:10.1038/nmeth.3932](http://dx.doi.org/10.1038/nmeth.3932)

Halsey LG, Curran-Everett D, Drummond GB. Response to “Confidence intervals are no salvation from the alleged fickleness of the P value.” Nat Meth. 2016. [doi:10.1038/nmeth.3933](http://dx.doi.org/10.1038/nmeth.3933)

Huber W. A clash of cultures in discussions of the P value. Nat Meth. 2016. [doi:10.1038/nmeth.3934](http://dx.doi.org/10.1038/nmeth.3934)

### Jul 28

Brodie A, Azaria JR, Ofran Y. How far from the SNP may the causative genes be? Nucleic Acids Research. 2016. [doi:10.1093/nar/gkw500](http://dx.doi.org/10.1093/nar/gkw500)

### Jul 27

Bewick AJ, Ji L, Niederhuth CE, Willing E-M, Hofmeister BT, Shi X, et al. On the origin and evolutionary consequences of gene body DNA methylation. PNAS. 2016. [doi:10.1073/pnas.1604666113](http:/dx.doi.org/10.1073/pnas.1604666113) 

### Jul 26

[http://www.johnmyleswhite.com/notebook/2016/07/23/why-im-not-a-fan-of-r-squared/](http://www.johnmyleswhite.com/notebook/2016/07/23/why-im-not-a-fan-of-r-squared/)

### Jul 25

Pedersen BS, Eyring K, De S, Yang IV, Schwartz DA. Fast and accurate alignment of long bisulfite-seq reads. arXiv. 2014. [arXiv:1401.1129](http://arxiv.org/abs/1401.1129)

### Jul 24

Gibson G. Rare and common variants: twenty arguments. Nature Reviews Genetics. 2012. [doi:10.1038/nrg3118](http://dx.doi.org/10.1038/nrg3118)

### Jul 23

Wang H-Q, Tuominen LK, Tsai C-J. SLIM: a sliding linear model for estimating the proportion of true null hypotheses in datasets with dependence structures. Bioinformatics. 2011. [doi:10.1093/bioinformatics/btq650](http://dx.doi.org/10.1093/bioinformatics/btq650)

### Jul 22

Goldman AD, Landweber LF. What Is a Genome? PLoS Genetics. 2016. [doi:10.1371/journal.pgen.1006181](http://dx.doi.org/10.1371/journal.pgen.1006181)

### Jul 21

Krzywinski M, Altman N. Analysis of variance andblocking. Nat Meth. 2014. [doi:10.1038/nmeth.3005](http://dx.doi.org/10.1038/nmeth.3005)

### Jul 20

Krzywinski M, Altman N. Nonparametric tests. Nat Meth. 2014. [doi:10.1038/nmeth.2937](http://dx.doi.org/10.1038/nmeth.2937)

### Jul 19

Krzywinski M, Altman N. Comparing samples—part II. Nat Meth. 2014. [doi:10.1038/nmeth.2900](http://dx.doi.org/10.1038/nmeth.2900)

### Jul 18

Krzywinski M, Altman N. Comparing samples—part I. Nat Meth. 2014. [doi:10.1038/nmeth.2858](http://dx.doi.org/10.1038/nmeth.2858)

### Jul 17

Grolemund G, Wickham H. A Cognitive Interpretation of Data Analysis. International Statistical Review. 2014. [doi:10.1111/insr.12028](http://dx.doi.org/10.1111/insr.12028)

### Jul 16

Ure K, Lu H, Wang W, Ito-Ishida A, Wu Z, He L-J, et al. Restoration of Mecp2 expression in GABAergic neurons is sufficient to rescue multiple disease features in a mouse model of Rett syndrome. eLife. 2016. [doi:10.7554/eLife.14198](http://dx.doi.org/10.7554/eLife.14198)

### Jul 15

Kawakatsu T, Huang S-SC, Jupe F, Sasaki E, Schmitz RJ, Urich MA, et al. Epigenomic Diversity in a Global Collection of Arabidopsis thaliana Accessions. Cell. 2016;166: 492–505. [doi:10.1016/j.cell.2016.06.044](http://dx.doi.org/10.1016/j.cell.2016.06.044)

### Jul 14

Iqbal K, Tran DA, Li AX, Warden C, Bai AY, Singh P, et al. Deleterious effects of endocrine disruptors are corrected in the mammalian germline by epigenome reprogramming. Genome Biology. 2015;16: 1–24. [doi:10.1186/s13059-015-0619-z](http://dx.doi.org/10.1186/s13059-015-0619-z)

### Jul 13

Teperek M, Simeone A, Gaggioli V, Miyamoto K, Allen GE, Erkek S, et al. Sperm is epigenetically programmed to regulate gene transcription in embryos. Genome Research. 2016. [doi:10.1101/gr.201541.115](http://dx.doi.org/10.1101/gr.201541.115)

### Jul 12

Meng D, Dubin M, Zhang P, Osborne EJ, Stegle O, Clark RM, et al. Limited Contribution of DNA Methylation Variation to Expression Regulation in Arabidopsis thaliana. PLoS Genetics. 2016;12: e1006141. [doi:10.1371/journal.pgen.1006141](http://dx.doi.org/10.1371/journal.pgen.1006141)

### Jul 11

McKiernan EC, Bourne PE, Brown CT, Buck S, Kenall A, Lin J, et al. How open science helps researchers succeed. eLife; 2016;5: e16800. [doi:10.7554/eLife.16800](http://dx.doi.org/10.7554/eLife.16800)

### Jul 10

Zhang Q, Wang D, Lang Z, He L, Yang L, Zeng L, et al. Methylation interactions in Arabidopsis hybrids require RNA-directed DNA methylation and are influenced by genetic variation. Proceedings of the National Academy of Sciences. 2016;: 201607851. [doi:10.1073/pnas.1607851113](http://dx.doi.org/10.1073/pnas.1607851113)

### Jul 9

del Olmo I, López JA, Vázquez J, Raynaud C, Piñeiro M, Jarillo JA. Arabidopsis DNA polymerase ϵ recruits components of Polycomb repressor complex to mediate epigenetic gene silencing. Nucleic Acids Research. 2016;44: gkw156–5614. [doi:10.1093/nar/gkw156](http://dx.doi.org/10.1093/nar/gkw156)

### Jul 8

Seymour M, Ji L, Santos AM, Kamei M, Sasaki T, Basenko EY, et al. Histone H1 Limits DNA Methylation in Neurospora crassa. G3. 2016. [doi:10.1534/g3.116.028324](http://dx.doi.org/10.1534/g3.116.028324)

### Jul 7

Chen X, Ge X, Wang J, Tan C, King GJ, Liu K. Genome-wide DNA methylation profiling by modified reduced representation bisulfite sequencing in Brassica rapa suggests that epigenetic modifications play a key role in polyploid genome evolution. Front Plant Sci. Frontiers; 2015;6: 1030. [doi:10.3389/fpls.2015.00836](http://dx.doi.org/10.3389/fpls.2015.00836)

### Jul 6

Yong W-S, Hsu F-M, Chen P-Y. Profiling genome-wide DNA methylation. Epigenetics & Chromatin. 2016;9: 1–16. [doi:10.1186/s13072-016-0075-3](http://dx.doi.org/10.1186/s13072-016-0075-3)

### Jul 5

Shewry PR, Pellny TK, Lovegrove A. Is modern wheat bad for health? Nature Plants. 2016;2: 16097. [doi:10.1038/nplants.2016.97](http://dx.doi.org/10.1038/nplants.2016.97)

### Jul 4

Lynch M. The frailty of adaptive hypotheses for the origins of organismal complexity. PNAS. 2007; [doi:10.1073/pnas.0702207104](http://dx.doi.org/10.1073/pnas.0702207104)

### Jul 3

Tufte, ER. Visual and Statistical Thinking: Displays of Evidence for Making Decisions.

### Jul 2

Lynch M, Conery JS. The origins of genome complexity. Science. 2003;302: 1401–1404. [doi:10.1126/science.1089370](http://dx.doi.org/10.1126/science.1089370)

Daubin V. Comment on “The Origins of Genome Complexity.” Science. 2004;306: 978a–978a. [doi:10.1126/science.1098469](http://dx.doi.org/10.1126/science.1098469)

Lynch M. Response to Comment on “The Origins of Genome Complexity.” Science. 2004;306: 978b–978b. [doi:10.1126/science.1100559](http://dx.doi.org/10.1126/science.1100559)

### Jul 1

Lever J, Krzywinski M, Altman N. Points of Significance: Logistic regression. Nat Meth. 13: 541–542. [doi:10.1038/nmeth.3904](http://dx.doi.org/10.1038/nmeth.3904)

### Jun 30

Zapata L, Ding J, Willing E-M, Hartwig B, Bezdan D, Jiao W-B, et al. Chromosome-level assembly of Arabidopsis thaliana Ler reveals the extent of translocation and inversion polymorphisms. Proceedings of the National Academy of Sciences. 2016. [doi: 10.1073/pnas.1607532113](http://dx.doi.org/10.1073/pnas.1607532113)

### Jun 29

Hutchins BI, Yuan X, Anderson JM, Santangelo GM. Relative Citation Ratio (RCR): A new metric that uses citation rates to measure influence at the article level. bioRxiv. 2015 Oct p. 029629. [doi:10.1101/029629](http://dx.doi.org/10.1101/029629)

### Jun 28

Libertini E, Heath SC, Hamoudi RA, Gut M, Ziller MJ, Herrero J, et al. Saturation analysis for whole-genome bisulfite sequencing data. Nat Biotech. [doi:10.1038/nbt.3524](http://dx.doi.org/10.1038/nbt.3524)

### Jun 27

Vale RD, Hyman AA. Priority of discovery in the life sciences. eLife. 2016;5: e16931. [doi:10.7554/eLife.16931](http://dx.doi.org/10.7554/eLife.16931)

### Jun 26

Davydov EV, Goode DL, Sirota M, Cooper GM, Sidow A, Batzoglou S. Identifying a High Fraction of the Human Genome to be under Selective Constraint Using GERP++. PLoS Comput Biol. 2010;6: e1001025. [doi:10.1371/journal.pcbi.1001025](http://dx.doi.org/10.1371/journal.pcbi.1001025)

### Jun 25

Morris ER, Grey H, McKenzie G, Jones AC, Richardson JM. A bend, flip and trap mechanism for transposon integration. eLife. 2016;5: e15537. [doi:10.7554/eLife.15537](http://dx.doi.org/10.7554/eLife.15537)

### Jun 24

Rahman R, Chirn G-W, Kanodia A, Sytnikova YA, Brembs B, Bergman CM, et al. Unique transposon landscapes are pervasive across Drosophila melanogaster genomes. Nucleic Acids Research. 2015;43: 10655–10672. [doi:10.1093/nar/gkv1193](http://dx.doi.org/10.1093/nar/gkv1193)

### Jun 23

Dolle D-D, Liu Z, Cotten ML, Simpson JT, Iqbal Z, Durbin R, et al. Using reference-free compressed data structures to analyse sequencing reads from thousands of human genomes. bioRxiv. 2016. [doi:10.1101/060186](http://dx.doi.org/10.1101/060186)

### Jun 22

Hough J, Williamson RJ, Wright SI. Patterns of selection in plant genomes. Annual Review of Ecology. 2013. [doi:10.1146/annurev-ecolsys-110512-135851](http://dx.doi.org/10.1146/annurev-ecolsys-110512-135851)

### Jun 21

Beissinger TM, Wang L, Crosby K, Durvasula A, Hufford MB, Ross-Ibarra J. Recent demography drives changes in linked selection across the maize genome. Nature Plants. 2: 16084 EP –. [doi:10.1038/nplants.2016.84](http://dx.doi.org/10.1038/nplants.2016.84)

### Jun 20

Platzer A, Nizhynska V, Long Q. TE-Locate: A Tool to Locate and Group Transposable Element Occurrences Using Paired-End Next-Generation Sequencing Data. Biology. 2012;1: 395–410. [doi:10.3390/biology1020395](http://dx.doi.org/10.3390/biology1020395)

### Jun 19

Huang W, Lyman RF, Lyman RA, Carbone MA, Harbison ST, Magwire MM, et al. Spontaneous mutations and the origin and maintenance of quantitative genetic variation. Przeworski M, editor. eLife. 2016;5: e14625. [doi:10.7554/eLife.14625](http://dx.doi.org/10.7554/eLife.14625)

### Jun 18

Poznik GD, Xue Y, Mendez FL, Willems TF, Massaia A, Wilson Sayres MA, et al. Punctuated bursts in human male demography inferred from 1,244 worldwide Y-chromosome sequences. Nat Genet. 48: 593–599. [doi:10.1038/ng.3559](http://dx.doi.org/10.1038/ng.3559)

### Jun 17

Ronan T, Qi Z, Naegle KM. Avoiding common pitfalls when clustering biological data. Science Signaling. 2016;9: re6–re6. [doi:10.1126/scisignal.aad1932](http://dx.doi.org/10.1126/scisignal.aad1932)

### Jun 16

Ellegren H, Galtier N. Determinants of genetic diversity. Nature Reviews Genetics. 17: 422–433. [doi:10.1038/nrg.2016.58](http://dx.doi.org/10.1038/nrg.2016.58)

### Jun 15

Wickham H. Tidy Data. Journal of Statistical Software. 2014;59: 1–23. [doi:10.18637/jss.v059.i10](http://dx.doi.org/10.18637/jss.v059.i10)

### Jun 14

Song M, Hao W, Storey JD. Testing for genetic associations in arbitrarily structured populations. Nat Genet. 47: 550–554. [doi:10.1038/ng.3244](http://dx.doi.org/10.1038/ng.3244)

### Jun 13

Wasserstein RL, Lazar NA. The ASA's Statement on p-Values: Context, Process, and Purpose. The American Statistician. 2016;70: 129–133. [doi:10.1080/00031305.2016.1154108](http://dx.doi.org/10.1080/00031305.2016.1154108)

Nuzzo R. Scientific method: statistical errors. Nature. 2014.: 150–152. [doi:10.1038/506150a](http://dx.doi.org/10.1038/506150a)

### Jun 12

Kass RE, Caffo BS, Davidian M, Meng X-L, Yu B, Reid N. Ten Simple Rules for Effective Statistical Practice. PLoS Comput Biol. 2016;12: e1004961. [doi:10.1371/journal.pcbi.1004961](http://dx.doi.org/10.1371/journal.pcbi.1004961)

### Jun 11

Jackel JN, Storer JM, Coursey T, Bisaro DM. Arabidopsis RNA polymerases IV and V are required to establish H3K9 methylation, but not cytosine methylation, on geminivirus chromatin. J Virol. 2016;: JVI.00656–16. [doi:10.1128/JVI.00656-16](http://dx.doi.org/10.1128/JVI.00656-16)

### Jun 10

Alonso-Blanco C, Andrade J, Becker C, Bemm F, Bergelson J, Borgwardt KM, et al. 1,135 Genomes Reveal the Global Pattern of Polymorphism in *Arabidopsis thaliana*. Cell. 2016. [doi:10.1016/j.cell.2016.05.063](http://dx.doi.org/10.1016/j.cell.2016.05.063)

### Jun 9

Schübeler D. Function and information content of DNA methylation. 2015;: 1–6. [doi:10.1038/nature14192](http://dx.doi.org/10.1038/nature14192)

### Jun 8

Hale CJ, Potok ME, Lopez J, Do T, Liu A, Gallego-Bartolome J, et al. Identification of Multiple Proteins Coupling Transcriptional Gene Silencing to Genome Stability in Arabidopsis thaliana. PLoS Genetics. 2016;12: e1006092. [doi:10.1371/journal.pgen.1006092](http://dx.doi.org/10.1371/journal.pgen.1006092)

### Jun 7

Beltran M, Yates CM, Skalska L, Dawson M, Reis FP, Viiri K, et al. The interaction of PRC2 with RNA or chromatin is mutually antagonistic. Genome Research. 2016. [doi:10.1101/gr.197632.115](http://dx.doi.org/10.1101/gr.197632.115)

### Jun 6

Chin C-S, Peluso P, Sedlazeck FJ, Nattestad M, Concepcion GT, Clum A, et al. Phased Diploid Genome Assembly with Single Molecule Real-Time Sequencing. bioRxiv. 2016 Jun p. 056887. [doi:10.1101/056887](http:/dx.doi.org/10.1101/056887)

### Jun 5

Quadrana L, Bortolini Silveira A, Mayhew GF, LeBlanc C, Martienssen RA, Jeddeloh JA, et al. The Arabidopsis thaliana mobilome and its impact at the species level. eLife. 2016;5: e15716. [doi:10.7554/eLife.15716](http://dx.doi.org/10.7554/eLife.15716)

### Jun 4

Taudt A, Colomé-Tatché M, Johannes F. Genetic sources of population epigenomic variation. Nature Reviews Genetics. 2016;: 1–14. [doi:10.1038/nrg.2016.45](http://dx.doi.org/10.1038/nrg.2016.45)

### Jun 3

Boeke JD, Church G, Hessel A, Kelley NJ, Arkin A, Cai Y, et al. The Genome Project–Write. Science. 2016. [doi:10.1126/science.aaf6850](http://dx.doi.org/10.1126/science.aaf6850)

### Jun 2

Pedersen BS, Layer RM, Quinlan AR. Vcfanno: fast, flexible annotation of genetic variants. Genome Biology. 2016;17: 1–9. [doi:10.1186/s13059-016-0973-5](http://dx.doi.org/10.1186/s13059-016-0973-5)

### Jun 1

Wibowo A, Becker C, Marconi G, Durr J, Price J, Hagmann J, et al. Hyperosmotic stress memory in Arabidopsis is mediated by distinct epigenetically labile sites in the genome and is restricted in the male germline by DNA glycosylase activity. eLife. 2016;5: e13546. [doi:10.7554/eLife.13546](http://dx.doi.org/10.7554/eLife.13546)

### May 31

Evrony GD, Lee E, Park PJ, Walsh CA. Resolving rates of mutation in the brain using single-neuron genomics. Eddy SR, editor. eLife. 2016;5: e12966. [doi:10.7554/eLife.12966](http://dx.doi.org/10.7554/eLife.12966)

### May 30

Hirsch CD, Springer NM. Transposable element influences on gene expression in plants. Biochim Biophys Acta. 2016. [doi:10.1016/j.bbagrm.2016.05.010](http://dx.doi.org/10.1016/j.bbagrm.2016.05.010)

### May 29

Li C, Gu L, Gao L, Chen C, Wei C-Q, Qiu Q, et al. Concerted genomic targeting of H3K27 demethylase REF6 and chromatin-remodeling ATPase BRM in Arabidopsis. Nature Genetics; 2016;48: 687–693. [doi:10.1038/ng.3555](http://dx.doi.org/10.1038/ng.3555)

### May 28

Cui X, Lu F, Qiu Q, Zhou B, Gu L, Zhang S, et al. REF6 recognizes a specific DNA sequence to demethylate H3K27me3 and regulate organ boundary formation in Arabidopsis. Nature Genetics; 2016;48: 694–699. [doi:10.1038/ng.3556](http://dx.doi.org/10.1038/ng.3556)

### May 27

Liu C, Wang C, Wang G, Becker C, Zaidem M, Weigel D. Genome-wide analysis of chromatin packing in Arabidopsis thaliana at single-gene resolution. Genome Research. 2016. [doi:10.1101/gr.204032.116](http://dx.doi.org/10.1101/gr.204032.116)

### May 26

Kim D-H, Sung S. The Plant Homeo Domain finger protein, VIN3-LIKE 2, is necessary for photoperiod-mediated epigenetic regulation of the floral repressor, MAF5. Proceedings of the National Academy of Sciences. 2010;107: 17029–17034. [doi:10.1073/pnas.1010834107](http://dx.doi.org/10.1073/pnas.1010834107)

### May 25

Greb T, Mylne JS, Crevillén P, Geraldo N, An H, Gendall AR, et al. The PHD finger protein VRN5 functions in the epigenetic silencing of Arabidopsis FLC. Current Biology. 2007;17: 73–78. [doi:10.1016/j.cub.2006.11.052](http://dx.doi.org/10.1016/j.cub.2006.11.052)

### May 24

Wang J, Yu Y, Tao F, Zhang J, Copetti D, Kudrna D, et al. DNA methylation changes facilitated evolution of genes derived from Mutator-like transposable elements. Genome Biology. 2016;: 1–20. [doi:10.1186/s13059-016-0954-8](http://dx.doi.org/10.1186/s13059-016-0954-8)

### May 23

Ahmed I, Sarazin A, Bowler C, Colot V, Quesneville H. Genome-wide evidence for local DNA methylation spreading from small RNA-targeted sequences in Arabidopsis. Nucleic Acids Research. 2011;39: 6919–6931. [doi:10.1093/nar/gkr324](http://dx.doi.org/10.1093/nar/gkr324)

### May 22

Shen H, He H, Li J, Chen W, Wang X, Guo L, et al. Genome-Wide Analysis of DNA Methylation and Gene Expression Changes in Two Arabidopsis Ecotypes and Their Reciprocal Hybrids. The Plant Cell. 2012;24: 875–892. [doi:10.1105/tpc.111.094870](http://dx.doi.org/10.1105/tpc.111.094870)

### May 21

Wang X, Weigel D, Smith LM. Transposon variants and their effects on gene expression in Arabidopsis. PLoS Genetics. 2013;9: e1003255. [doi:10.1371/journal.pgen.1003255](http://dx.doi.org/10.1371/journal.pgen.1003255)

### May 20

O'Malley RC, Huang S-SC, Song L, Lewsey MG, Bartlett A, Nery JR, et al. Cistrome and Epicistrome Features Shape the Regulatory DNA Landscape. Cell. 2016;165: 1280–1292. [doi:10.1016/j.cell.2016.04.038](http://dx.doi.org/10.1016/j.cell.2016.04.038)

### May 19

Moreno-Romero J, Jiang H, Santos-González J, Köhler C. Parental epigenetic asymmetry of PRC2-mediated histone modifications in the Arabidopsis endosperm. EMBO. 2016;: 1–14. [doi:10.15252/embj.201593534](http://dx.doi.org/10.15252/embj.201593534)

### May 18

Zuk O, Schaffner SF, Samocha K, Do R, Hechter E, Kathiresan S, et al. Searching for missing heritability: Designing rare variant association studies. Proc Natl Acad Sci USA. 2014; 111: E455–E464. [doi:10.1073/pnas.1322563111](http://dx.doi.org/10.1073/pnas.1322563111)

### May 17

Zuk O, Hechter E, Sunyaev SR, Lander ES. The mystery of missing heritability: Genetic interactions create phantom heritability. Proc Natl Acad Sci USA. 2012;109: 1193–1198. [doi:10.1073/pnas.1119675109](http://dx.doi.org/10.1073/pnas.1119675109)

### May 16

Sasaki E, Zhang P, Atwell S, Meng D, Nordborg M. “Missing” G x E Variation Controls Flowering Time in Arabidopsis thaliana. PLoS Genetics; 2015;11: e1005597. [doi:10.1371/journal.pgen.1005597](http://dx.doi.org/10.1371/journal.pgen.1005597)

### May 15

Horton MW, Bodenhausen N, Beilsmith K, Meng D, Muegge BD, Subramanian S, et al. Genome-wide association study of Arabidopsis thaliana leaf microbial community. Nature Communications; 2014;5: 1–7. [doi:10.1038/ncomms6320](http://dx.doi.org/10.1038/ncomms6320)

### May 14

Sudmant PH, Rausch T, Gardner EJ, Handsaker RE, Abyzov A, Huddleston J, et al. An integrated map of structural variation in 2,504 human genomes. Nature. 2015;526: 75–81. [doi:10.1038/nature15394](http://dx.doi.org/10.1038/nature15394)

### May 13

Ewing AD, Kazazian HH. Whole-genome resequencing allows detection of many rare LINE-1 insertion alleles in humans. Genome Research. 2011;21: 985–990. [doi:10.1101/gr.114777.110](http://dx.doi.org/10.1101/gr.114777.110)

### May 12

Ewing AD, Kazazian HH. High-throughput sequencing reveals extensive variation in human-specific L1 content in individual human genomes. Genome Research. 2010;20: 1262–1270. [doi:10.1101/gr.106419.110](http://dx.doi.org/10.1101/gr.106419.110)

### May 11

Montgomery SB, Lappalainen T, Gutierrez-Arcelus M, Dermitzakis ET. Rare and Common Regulatory Variation in Population-Scale Sequenced Human Genomes. PLoS Genetics; 2011;7: e1002144–10. [doi:10.1371/journal.pgen.1002144](http://dx.doi.org/10.1371/journal.pgen.1002144)

### May 10

Piegu B, Guyot R, Picault N, Roulin A, Sanyal A, Saniyal A, et al. Doubling genome size without polyploidization: dynamics of retrotransposition-driven genomic expansions in Oryza australiensis, a wild relative of rice. Genome Research. 2006;16: 1262–1269. [doi:10.1101/gr.5290206](http://dx.doi.org/10.1101/gr.5290206)

### May 9

Morris AP, Zeggini E. An evaluation of statistical approaches to rare variant analysis in genetic association studies. Genet Epidemiol. 2009;34: 188–193. [doi:10.1002/gepi.20450](http://dx.doi.org/10.1002/gepi.20450)

### May 8

Flutre T, Duprat E, Feuillet C, Quesneville H. Considering Transposable Element Diversification in De Novo Annotation Approaches. PLoS ONE. 2011;6: e16526. [doi:10.1371/journal.pone.0016526](http:/dx.doi.org/10.1371/journal.pone.0016526)

### May 7

Hunt KA, Mistry V, Bockett NA, Ahmad T, Ban M, Barker JN, et al. Negligible impact of rare autoimmune-locus coding-region variants on missing heritability. Nature; 498: 232–235. [doi:10.1038/nature12170](http://dx.doi.org/10.1038/nature12170)

### May 6

Walker AR, Lee E, Bogs J, McDavid DAJ, Thomas MR, Robinson SP. White grapes arose through the mutation of two similar and adjacent regulatory genes. The Plant Journal. 2007;49: 772–785. [doi:10.1111/j.1365-313X.2006.02997.x](http://dx.doi.org/10.1111/j.1365-313X.2006.02997.x)

### May 5

Pelsy F, Dumas V, Bévilacqua L, Hocquigny S, Merdinoglu D. Chromosome Replacement and Deletion Lead to Clonal Polymorphism of Berry Color in Grapevine. PLoS Genetics. 2015;11: e1005081. [doi:10.1371/journal.pgen.1005081](http://dx.doi.org/10.1371/journal.pgen.1005081)

### May 4

Teng M, Love MI, Davis CA, Djebali S, Dobin A, Graveley BR, et al. A benchmark for RNA-seq quantification pipelines. Genome Biology. 2016;17: 636–12. [doi:10.1186/s13059-016-0940-1](http://dx.doi.org/10.1186/s13059-016-0940-1)

### May 3

Zhao J, Akinsanmi I, Arafat D, Cradick TJ, Lee CM, Banskota S, et al. A Burden of Rare Variants Associated with Extremes of Gene Expression in Human Peripheral Blood. The American Journal of Human Genetics. 2016;98: 299–309. [doi:10.1016/j.ajhg.2015.12.023](http://dx.doi.org/10.1016/j.ajhg.2015.12.023)

### May 2

Li YI, van de Geijn B, Raj A, Knowles DA, Petti AA, Golan D, et al. RNA splicing is a primary link between genetic variation and disease. Science. American Association for the Advancement of Science; 2016;352: 600–604. [doi:10.1126/science.aad9417](http://dx.doi.org/10.1126/science.aad9417)

### May 1

Maumus, F. & Quesneville, H. Ancestral repeats have shaped epigenome and genome composition for millions of years in Arabidopsis thaliana. Nature Communications 5, (2014). [doi:10.1038/ncomms5104](http://dx.doi.org/10.1038/ncomms5104)

### Apr 30

Altman, N. & Krzywinski, M. Points of Significance: Regression diagnostics. Nat Meth 13, 385–386. [doi:10.1038/nmeth.3854](http://dx.doi.org/10.1038/nmeth.3854)

### Apr 29

Baidouri El M, Kim KD, Abernathy B, Arikit S, Maumus F, Panaud O, et al. A new approach for annotation of transposable elements using small RNA mapping. Nucleic Acids Research. 2015;43: e84–e84. [doi:10.1093/nar/gkv257](http://dx.doi.org/10.1093/nar/gkv257)

### Apr 28

Caspi A, Pachter L. Identification of transposable elements using multiple alignments of related genomes. Genome Research. 2006;16: 260–270. [doi:10.1101/gr.4361206](http://dx.doi/10.1101/gr.4361206)

### Apr 27

Flutre T, Permal E, Quesneville H. Transposable element annotation in completely sequenced eukaryote genomes. Plant Transposable Elements. Springer; 2012. pp. 17–39. [ISBN: 978-3-642-31841-2](http://link.springer.com/chapter/10.1007/978-3-642-31842-9_2)

### Apr 26

Exposito-Alonso M, Becker C, Schuenemann VJ, Reitter E, Setzer C, Slovak R, et al. The rate and effect of de novo mutations in natural populations of Arabidopsis thaliana. bioRxiv. 2016. [doi:10.1101/050203](http://dx.doi.org/10.1101/050203)

### Apr 25

Hoen DR, Hickey G, Bourque G, Casacuberta J, Cordaux R, Feschotte C, et al. A call for benchmarking transposable element annotation methods. Mobile DNA. 2015;6: 1–9. [doi:10.1186/s13100-015-0044-6](http://dx.doi.org/10.1186/s13100-015-0044-6)

### Apr 24

Sanjana NE, Wright J, Zheng K, Shalem O, Fontanillas P, Joung J, et al. High-resolution interrogation of functional elements in the noncoding genome. bioRxiv. 2016 Apr p. 049130. [doi:10.1101/049130](http://dx.doi.org/10.1101/049130)

### Apr 23

Conroy RM. What hypotheses do “nonparametric” two-group tests actually test? Stata Journal. College Station, TX: Stata Press; [2012;12: 182–190.](http://www.stata-journal.com/article.html?article=st0253)

### Apr 22

Narasimhan VM, Hunt KA, Mason D, Baker CL, Karczewski KJ, Barnes MR, et al. Health and population effects of rare gene knockouts in adult humans with related parents. Science. 2016;352: 474–477. [doi:10.1126/science.aac8624](http://science.sciencemag.org/content/352/6284/474)

### Apr 21

Bergelson J, Buckler ES, Ecker JR, Nordborg M, Weigel D. A Proposal Regarding Best Practices for Validating the Identity of Genetic Stocks and the Effects of Genetic Variants. The Plant Cell. 2016;28: 606–609. [doi:10.1105/tpc.15.00502](http://www.plantcell.org/content/28/3/606.long)

### Apr 20

Krzywinski M, Altman N. Importance of being uncertain. Nature Methods. 2013;10: 809–810. [doi:10.1038/nmeth.2613](http://www.nature.com/nmeth/journal/v10/n9/full/nmeth.2613.html)

### Apr 19

Zhang Z, Liu X, Guo X, Wang X-J, Zhang X. Arabidopsis AGO3 predominantly recruits 24-nt small RNAs to regulate epigenetic silencing. Nature Plants. [doi:10.1038/nplants.2016.49](http://www.nature.com/articles/nplants201649)

### Apr 18

Diehl WE, Patel N, Halm K, Johnson WE. Tracking interspecies transmission and long-term evolution of an ancient retrovirus using the genomes of modern mammals. Goff SP, editor. eLife. 2016;5: e12704. [doi:10.7554/eLife.12704](http://elifesciences.org/content/5/e12704v1)

### Apr 17

Chen R, Shi L, Hakenberg J, Naughton B, Sklar P, Zhang J, et al. Analysis of 589,306 genomes identifies individuals resilient to severe Mendelian childhood diseases. Nature Biotechnology. [doi:10.1038/nbt.3514](http://www.nature.com/nbt/journal/vaop/ncurrent/full/nbt.3514.html)

### Apr 16

Oikonomopoulos S, Wang YC, Djambazian H, Badescu D, Ragoussis J. Benchmarking of the Oxford Nanopore MinION sequencing for quantitative and qualitative assessment of cDNA populations. bioRxiv. 2016 Apr p. 048074. [doi:10.1101/048074](http://biorxiv.org/content/early/2016/04/11/048074)

### Apr 15

Jeffares DC, Jolly C, Hoti M, Speed D, Rallis C, Dessimoz C, et al. Transient structural variations alter gene expression and quantitative traits in Schizosaccharomyces pombe. bioRxiv. 2016 Apr p. 047266. [doi:10.1101/047266](http://biorxiv.org/content/biorxiv/early/2016/04/13/047266)

### Apr 14

Linheiro RS, Bergman CM. 2012. Whole Genome Resequencing Reveals Natural Target Site Preferences of Transposable Elements in Drosophila melanogaster ed. J.E. Stajich. PLoS ONE 7: e30008. [doi: 10.1371/journal.pone.0030008](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0030008)

### Apr 13

Wilson G. Software Carpentry: lessons learned [version 2; referees: 3 approved]. F1000Research 2016, 3:62 [doi: 10.12688/f1000research.3-62.v2](http://f1000research.com/articles/3-62/v2)

### Apr 12

Lee HC, Lai K, Lorenc MT, Imelfort M, Duran C, Edwards D. 2011. Bioinformatics tools and databases for analysis of next-generation sequence data. Briefings in Functional Genomics. [doi:10.1093/bfgp/elr037](http://bfg.oxfordjournals.org/content/early/2011/12/18/bfgp.elr037.full)

### Apr 11

Muñoz-Lopez M, Garcia-Perez JL. 2010. DNA Transposons: Nature and Applications in Genomics. Curr Genomics 11: 115–128. [doi: 10.2174/138920210790886871](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2874221/)

### Apr 10

Vourekas A, Alexiou P, Vrettos N, Maragkakis M, Mourelatos Z. 2016. Sequence-dependent but not sequence-specific piRNA adhesion traps mRNAs to the germ plasm. Nature 1–24. [doi:10.1038/nature17150](http://www.nature.com/nature/journal/v531/n7594/full/nature17150.html)

### Apr 9

Kanno S, Arrighi J-F, Chiarenza S, Bayle V, Berthomé R, Péret B, Javot H, Delannoy E, Marin E, Nakanishi TM, et al. 2016. A novel role for the root cap in phosphate uptake and homeostasis ed. D. Weigel. eLife 5: e14577. [doi:10.7554/eLife.14577](http://elifesciences.org/content/5/e14577v1)

### Apr 8

Calderwood A, Kopriva S, Morris RJ. 2016. Transcript Abundance Explains mRNA Mobility Data in Arabidopsis thaliana. The Plant Cell 28: 610–615. [doi:10.1105/tpc.15.00956](http://www.plantcell.org/content/28/3/610)

### Apr 7

Bewick AJ, Ji L, Niederhuth CE, Willing E-M, Hofmeister BT, Shi X, Wang L, Lu Z, Rohr NA, Hartwig B, et al. 2016. On the Origin and Evolutionary Consequences of Gene Body DNA Methylation. bioRxiv. [doi:10.1101/045542](http://biorxiv.org/content/biorxiv/early/2016/03/24/045542)

### Apr 6

Simpson JT, Workman R, Zuzarte PC, David M, Dursi LJ, Timp W. 2016. Detecting DNA Methylation using the Oxford Nanopore Technologies MinION sequencer. bioRxiv. [doi: 10.1101/047142](http://biorxiv.org/content/biorxiv/early/2016/04/04/047142)

Rand AC, Jain M, Eizenga J, Musselman-Brown A, Olsen HE, Akeson M, Paten B. 2016. Cytosine Variant Calling with High-throughput Nanopore Sequencing. bioRxiv [doi:10.1101/047134](http://biorxiv.org/content/biorxiv/early/2016/04/04/047134)

### Apr 5

Toomajian C, Ajioka RS, Jorde LB, Kushner JP, Kreitman M. 2003. A method for detecting recent selection in the human genome from allele age estimates. Genetics 165: 287–297. [PMC1462736](http://www.ncbi.nlm.nih.gov/pubmed/14504236)

### Apr 4

Genin E, Tullio-Pelet A, Begeot F, Lyonnet S, Abel L. 2004. Estimating the age of rare disease mutations: the example of Triple-A syndrome. Journal of Medical Genetics 41: 445–449. [doi:10.1136/jmg.2003.017962](http://jmg.bmj.com/content/41/6/445.full)

### Apr 3

Slatkin M. 2008. Linkage disequilibrium — understanding the evolutionary past and mapping the medical future. Nature Reviews Genetics 9: 477–485. [doi:10.1038/nrg2361](http://www.nature.com/nrg/journal/v9/n6/full/nrg2361.html)

### Apr 2

Ben-Moshe S, Itzkovitz S. 2016. Bursting through the cell cycle. eLife 5: e14953. [doi:10.7554/eLife.14953](http://elifesciences.org/content/5/e14953v1)

### Apr 1

Bourne HR. 2013. The writing on the wall. eLife 2: e00642. [doi: 10.7554/eLife.00642](http://elifesciences.org/content/2/e00642v1)

### Mar 31

McClintock B. 1950. The origin and behavior of mutable loci in maize. Proc Natl Acad Sci USA 36: 344–355. [PMC1063197](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1063197/)

### Mar 30

Berry S, Hartley M, Olsson TSG, Dean C, Howard M. 2015. Local chromatin environment of a Polycomb target gene instructs its own epigenetic inheritance ed. D. Zilberman. eLife 4: e07205. [doi:10.7554/eLife.07205](http://elifesciences.org/content/4/e07205v2)

### Mar 29

Sánchez-Gracia A, Maside X, Charlesworth B. High rate of horizontal transfer of transposable elements in Drosophila. Trends in Genetics 21: 200–203. [doi:10.1016/j.tig.2005.02.001](http://www.sciencedirect.com/science/article/pii/S0168952505000417)

### Mar 28

Hill T, Schlötterer C, Betancourt AJ. 2016. Hybrid Dysgenesis in Drosophila simulans Associated with a Rapid Invasion of the P-Element ed. H.S. Malik. PLoS Genetics 12: e1005920. [doi: 10.1371/journal.pgen.1005920](http://journals.plos.org/plosgenetics/article?id=10.1371%2Fjournal.pgen.1005920&utm_source=feedburner&utm_medium=feed&utm_campaign=Feed%3A+plosgenetics%2FNewArticles+%28PLOS+Genetics+-+New+Articles%29)

### Mar 27

Greaves IK, Groszmann M, Ying H, Taylor JM, Peacock WJ, Dennis ES. 2012. Trans Chromosomal Methylation in Arabidopsis hybrids. Proc Natl Acad Sci USA 109: 3570–3575. [doi:10.1073/pnas.1201043109](http://www.pnas.org/content/109/9/3570.full)

### Mar 26

Rigal M, Becker C, Pélissier T, Pogorelcnik R, Devos J, Ikeda Y, Weigel D, Mathieu O. 2016. Epigenome confrontation triggers immediate reprogramming of DNA methylation and transposon silencing in Arabidopsis thaliana F1 epihybrids. Proceedings of the National Academy of Sciences 201600672–10. [doi: 10.1073/pnas.1600672113](http://www.pnas.org/content/early/2016/03/16/1600672113.full)

### Mar 25

Groszmann M, Greaves IK, Albertyn ZI, Scofield GN, Peacock WJ, Dennis ES. 2011. Changes in 24-nt siRNA levels in Arabidopsis hybrids suggest an epigenetic contribution to hybrid vigor. Proceedings of the National Academy of Sciences 108: 2617–2622. [doi: 10.1073/pnas.1019217108](http://www.pnas.org/content/108/6/2617.full)

### Mar 24

Fang X, Qi Y. 2016. RNAi in Plants: An Argonaute-Centered View. Plant Cell. Advance Publication February 11, 2016. [doi: 10.​1105/​tpc.​15.​00920](http://www.plantcell.org/content/early/2016/02/11/tpc.15.00920.abstract)

### Mar 23

Miga KH, Eisenhart C, Kent WJ. 2015. Utilizing mapping targets of sequences underrepresented in the reference assembly to reduce false positive alignments. Nucleic Acids Research 43: e133–8. [doi: 10.1093/nar/gkv671](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4787761/)

### Mar 22

Martínez G, Panda K, Köhler C, Slotkin RK. 2016. Silencing in sperm cells is directed by RNA movement from the surrounding nurse cell. Nature Plants 1–8. [DOI: 10.1038/NPLANTS.2016.30](http://www.nature.com/articles/nplants201630)

### Mar 21

Yamamuro C, Miki D, Zheng Z, Ma J, Wang J, Yang Z, Dong J, Zhu J-K. 1AD. Overproduction of stomatal lineage cells in Arabidopsis mutants defective in active DNA demethylation. Nature Communications 5: 1–7. [DOI: 10.1038/ncomms5062](http://www.nature.com/ncomms/2014/140605/ncomms5062/full/ncomms5062.html)

### Mar 20

Saito Y, Tsuji J, Mituyama T. 2014. Bisulfighter: accurate detection of methylated cytosines and differentially methylated regions. Nucleic Acids Research. [doi:10.1093/nar/gkt1373](http://www.ncbi.nlm.nih.gov/pubmed/24423865)

### Mar 19

Guo W, Fiziev P, Yan W, Cokus S, Sun X, Zhang MQ, Chen P-Y, Pellegrini M. 2013. BS-Seeker2: a versatile aligning pipeline for bisulfite sequencing data. BMC Genomics 14: 774. [DOI: 10.1186/1471-2164-14-774](http://bmcgenomics.biomedcentral.com/articles/10.1186/1471-2164-14-774)

### Mar 18

Elvira-Matelot E, Hachet M, Shamandi N, Comella P, Saez-Vásquez J, Zytnicki M, Vaucheret H. 2016. Arabidopsis RNASE THREE LIKE2 Modulates the Expression of Protein-Coding Genes via 24-Nucleotide Small Interfering RNA-Directed DNA Methylation. The Plant Cell 28: 406–425. [doi:10.1105/tpc.15.00540](http://www.plantcell.org/content/28/2/406.short?rss=1)

### Mar 17

Adrian J, Chang J, Ballenger CE, Bargmann BOR, Alassimone J, Davies KA, Lau OS, Matos JL, Hachez C, Lanctot A, et al. 2015. Transcriptome Dynamics of the Stomatal Lineage: Birth, Amplification, and Termination of a Self- Renewing Population. Dev Cell 33: 107–118. [doi:10.1016/j.devcel.2015.01.025](http://www.sciencedirect.com/science/article/pii/S1534580715000684)

### Mar 16

Maumus F, Quesneville H. 2016. Impact and insights from ancient repetitive elements in plant genomes. Current Opinion in Plant Biology 30: 41–46. [doi:10.1016/j.pbi.2016.01.003](http://www.sciencedirect.com/science/article/pii/S1369526616300036)

### Mar 15

Grandbastien MA, Audeon C, Bonnivard E, Casacuberta JM, Chalhoub B, Costa APP, Le QH, Melayah D, Petit M, Poncet C, et al. 2005. Stress activation and genomic impact of Tnt1 retrotransposons in Solanaceae. Cytogenet Genome Res 110: 229–241. [DOI: 10.1159/000084957](http://www.ncbi.nlm.nih.gov/pubmed/16093677)

### Mar 14

Wolff P, Jiang H, Wang G, Santos-González J, Köhler C. 2015. Paternally expressed imprinted genes establish postzygotic hybridization barriers in Arabidopsis thaliana. eLife. [DOI: 10.7554/eLife.10074](http://elifesciences.org/content/4/e10074v2#api_box)

### Mar 13

Weiß CL, Dannemann M, Prüfer K, Burbano HA. 2015. Contesting the presence of wheat in the British Isles 8,000 years ago by assessing ancient DNA authenticity from low-coverage data ed. J.K. Pickrell. eLife 4: e10005. [DOI: 10.7554/eLife.10005](http://elifesciences.org/content/4/e10005v1)

### Mar 12

Loose M, Malla S, Stout M. 2016. Real time selective sequencing using nanopore technology. bioRxiv 038760. [doi: 10.1101/038760](http://biorxiv.org/content/early/2016/02/03/038760)

### Mar 11

Mita P, Boeke JD. 2016. How retrotransposons shape genome regulation. Current Opinion in Genetics & Development 37: 90–100. [doi:10.1016/j.gde.2016.01.001](http://www.sciencedirect.com/science/article/pii/S0959437X16000022)

### Mar 10

O'Neill RJW, O'Neill MJ, Graves JAM. 1998. Undermethylation associated with retroelement activation and chromosome remodelling in an interspecific mammalian hybrid. Nature 393: 68–72. [doi:10.1038/29985](http://www.nature.com/doifinder/10.1038/29985)

### Mar 9

Zeh DW, Zeh JA, Ishida Y. 2009. Transposable elements and an epigenetic basis for punctuated equilibria. BioEssays 31: 715–726. [doi:10.1002/bies.200900026](http://onlinelibrary.wiley.com/enhanced/doi/10.1002/bies.200900026)

### Mar 8

Rosenthal R, Fode KL. 2007. The effect of experimenter bias on the performance of the albino rat. Syst Res 8: 183–189. [doi:10.1002/bs.3830080302](http://onlinelibrary.wiley.com/doi/10.1002/bs.3830080302/abstract)

### Mar 7

Rivas MA, Graham D, Sulem P, Stevens C, Desch N. 2015. A protein truncating R179X variant in RNF186 confers protection against ulcerative colitis. bioRxiv. [doi:10.1101/035105](http://dx.doi.org/10.1101/035105)

### Mar 6

Yuan K, O'Farrell PH. 2016. TALE-light imaging reveals maternally guided, H3K9me2/3-independent emergence of functional heterochromatin in Drosophila embryos. Genes & Development 30: 579–593. [doi:10.1101/gad.272237.115.](http://genesdev.cshlp.org/content/30/5/579.short?rss=1)

### Mar 5

Sul JH, Bilow M, Yang W-Y, Kostem E, Furlotte N, He D, Eskin E. 2016. Accounting for Population Structure in Gene-by-Environment Interactions in Genome-Wide Association Studies Using Mixed Models ed. N.J. Schork. PLoS Genetics 12: e1005849–19. [DOI: 10.1371/journal.pgen.1005849](http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1005849)

### Mar 4

Hsi-Yang Fritz M, Leinonen R, Cochrane G, Birney E. 2011. Efficient storage of high throughput DNA sequencing data using reference-based compression. Genome Research 21: 734–740. [doi:10.1101/gr.114819.110.](http://genome.cshlp.org/content/21/5/734.long)

### Mar 3

Levasseur A, Bekliz M, Chabrière E, Pontarotti P, La Scola B, Raoult D. 2016. MIMIVIRE is a defence system in mimivirus that confers resistance to virophage. Nature 1–12. [doi:10.1038/nature17146](http://www.nature.com/nature/journal/vaop/ncurrent/full/nature17146.html)

### Mar 2

Bogdanovic O, Smits AH, la Calle Mustienes de E, Tena JJ, Ford E, Williams R, Senanayake U, Schultz MD, Hontelez S, van Kruijsbergen I, et al. 2016. Active DNA demethylation at enhancers during the vertebrate phylotypic period. Nat Genet 1–13. [doi:10.1038/ng.3522](http://dx.doi.org/10.1038/ng.3522)

### Mar 1

Lewsey MG, Hardcastle TJ, Melnyk CW, Molnar A, Valli A, Urich MA, Nery JR, Baulcombe DC, Ecker JR. 2016. Mobile small RNAs regulate genome-wide DNA methylation. Proceedings of the National Academy of Sciences. [doi:10.1073/pnas.1515072113](http://www.pnas.org/content/113/6/E801.abstract)

### Feb 29

Storey JD, Tibshirani R. 2003. Statistical significance for genomewide studies. Proc Natl Acad Sci USA 100: 9440–9445. [doi:10.1073/pnas.1530509100](http://www.pnas.org/content/100/16/9440.full)

### Feb 28

Fernandez AG, Mis EK, Bargmann BOR, Birnbaum KD, Piano F. 2010. Automated sorting of live C. elegans using laFACS. Nat Meth 7: 417–418. [doi:10.1038/nmeth.f.304.](http://www.nature.com/nmeth/journal/v7/n6/full/nmeth.f.304.html)

### Feb 27

Slaughter A, Daniel X, Flors V, Luna E, Hohn B, Mauch-Mani B. 2012. Descendants of Primed Arabidopsis Plants Exhibit Resistance to Biotic Stress. Plant Physiology 158: 835–843. [doi:​10.​1104/​pp.​111.​191593](http:/​/​dx.​doi.​org/​10.​1104/​pp.​111.​191593)

### Feb 26

Guerreiro MPG. 2014. Interspecific hybridization as a genomic stressor inducing mobilization of transposable elements in Drosophila. Mob Genet Elements 4: e34394. [doi.org/10.4161/mge.34394](http://dx.doi.org/10.4161/mge.34394)

### Feb 25

Vale RD. 2015. Accelerating Scientific Publication in Biology. bioRxiv 022368. [doi:10.1101/022368](http://biorxiv.org/content/early/2015/09/12/022368)

> *Related:*
> [http://www.michaeleisen.org/blog/?p=1733](http://www.michaeleisen.org/blog/?p=1733)


### Feb 24

Clavel M, Pélissier T, Descombin J, Jean V, Picart C, Charbonel C, Saez-Vásquez J, Bousquet-Antonelli C, Deragon J-M. 2015. Parallel action of AtDRB2 and RdDM in the control of transposable element expression. BMC Plant Biology 15: 70. [DOI 10.1186/s12870-015-0455-z](http://bmcplantbiol.biomedcentral.com/articles/10.1186/s12870-015-0455-z)


### Feb 23

Crisp PA, Ganguly D, Eichten SR, Borevitz JO, Pogson BJ. 2016. Reconsidering plant memory: Intersections between stress recovery, RNA turnover, and epigenetics. Science Advances 2: e1501340–e1501340. [doi:10.1126/sciadv.1501340](http://advances.sciencemag.org/content/advances/2/2/e1501340.full)

*Interesting or important points:*  

> An as yet entirely unexplored possibility is the prospect that transcriptional memory might be underpinned by changes in mRNA stability rather than transcription. 

I think certain RNA modifications have been shown to be induced by stress (at least in yeast), perhaps they play a role in modifying mRNA stability to facilitate stress memory.
